core/arch/x86/disassemble.c

```diff
@@ -195,6 +195,7 @@ opnd_disassemble_noimplicit(char *buf, size_t bufsz, size_t *sofar INOUT,
     case TYPE_FLOATMEM:
```

Comment:
Please add a test to ir_x86.c where we have some disassembly output string comparisons.

Reply:
I am adding an entire suite of tests right now along with #3574, these opcodes here will be included. It will take a little bit as I am going through the failures one by one. Re-open if necessary.

Reply:
OK, though we like to have corner cases or past bugs included in our own tests, which includes this.

Reply:
I have added a small dissembler test to ir_x86.